PWGHF/D2H/Tasks/taskLc.cxx

```diff
@@ -452,8 +551,27 @@ struct HfTaskLc {
           registry.fill(HIST("MC/reconstructed/nonprompt/hImpParErrProng2SigNonPrompt"), candidate.errorImpactParameter2(), pt);
           registry.fill(HIST("MC/reconstructed/nonprompt/hDecLenErrSigNonPrompt"), candidate.errorDecayLength(), pt);
         }
+        if constexpr (applyMl) {
+          double Mass(-1);
+          double outputBkg(-1);
+          double outputFD(-1);
+          if ((candidate.isSelLcToPKPi() >= selectionFlagLc) && pdgCodeProng0 == kProton) {
+            Mass = hfHelper.invMassLcToPKPi(candidate);
+            outputBkg = candidate.mlProbLcToPKPi()[0]; /// bkg score
+            outputFD = candidate.mlProbLcToPKPi()[1];  /// non-prompt score
```

Comment:
Typically we put the FD score in 3rd position (see comment above)
Also same comment about possibly empty vectors (see above)